packages/compiler/test/parser.test.ts

```diff
@@ -209,15 +209,11 @@ describe("compiler: parser", () => {
       "@myDec union A { @myDec a: string }",
       "union A<T, V> { a: T; none: {} }",
       `union A { "hi there": string }`,
+      `union A { string, int32 }`,
+      `union A { B<T>, C<T> }`,
+      `union A { "hi", \`bye\` }`,
     ]);
-
-    parseErrorEach([
-      [
-        'union A { @myDec "x" x: number, y: string }',
-        [/':' expected/],
-        (n) => assert(!n.printable, "should not be printable"),
```

Comment:
The `should not be printable` assertion I think no longer applies here? The parser now interprets the above as missing a union variant list delimiter between the anonymous variant "x" and the following named variant, and we have said that trees are printable if the only thing missing is a delimiter.